### PR TITLE
Html mail

### PR DESCRIPTION
**Pull request by jaolejnik**

### CODE REVIEW

File: client/src/components/template/templateCard.component.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ import React, { useEffect } from "react";
 import { Link } from "react-router-dom";
 import M from "materialize-css";
 
-String.prototype.trunc =
-  String.prototype.trunc ||
-  function(n) {
-    return this.length > n ? this.substr(0, n - 1) + "&hellip;" : this;
-  };
+// String.prototype.trunc =
```
Comment (starmarek) [inline client/src/components/template/templateCard.component.js:6]: You should remove commented code when you deliver to master.

File: client/src/components/template/templateDashboard.component.js [hunk 2]
```diff
@@ -18,24 +18,11 @@ const TemplateDashboard = () => {
...
-        tmp.push("col s12 m6");
-      }
-    });
-    setLayout(tmp);
-  };
 
   useEffect(() => {
     // init materialize css components
     let elems = document.querySelectorAll(".fixed-action-btn");
-    let instances = M.FloatingActionButton.init(elems, {
```
Comment (starmarek) [inline client/src/components/template/templateDashboard.component.js:25]: Love that.

File: client/src/components/template/templateCard.component.js [hunk 3]
```diff
@@ -13,7 +14,6 @@ const sendInv = invID => {
 };
 
 const setCardIcon = trainingType => {
-  let cardColor;
```
Comment (starmarek) [inline client/src/components/template/templateCard.component.js:16]: And that also.

File: client/src/components/template/templateCard.component.js (hunk 1 above)
Comment (jaolejnik) [reply client/src/components/template/templateCard.component.js:6]: Noted.

**Comment by jaolejnik** [comment]
> Also, you should really work on your git commit messages.
> _a bit better, but still bad_ is a total rubbish.
I completely agree. It was one time thing.
> You can try to amend it (rename) in order to improve git skills.
Exactly what I've done.

**Pull request merged**